### PR TITLE
Feature/fetch ranking data

### PR DESCRIPTION
**Pull request by TomYYHong**
Added the fetch data function in Ranking page

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const App = () => {
                 <Route path="auth" element={<AuthPage/>}/>
                 <Route element={<Navigation/>}>
                     <Route index element={<HomePage/>}/>
-                    <Route path="activities" element={<ActivitiesPage/>}/>
+                    {/* <Route path="activities" element={<ActivitiesPage/>}/> */}
```
Comment (JordanLDavid) [inline src/App.tsx]: kindly delete instead of commenting out

File: src/components/Post/Post.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import "./Post.css";
 import {Avatar, Paper} from '@mui/material';
-import {deepPurple} from '@mui/material/colors';
+import { deepPurple} from '@mui/material/colors';
 // import ThumbUpIcon from '@mui/icons-material/ThumbUp';
```
Comment (JordanLDavid) [inline src/components/Post/Post.tsx]: remove unnecessary icons

File: src/components/Post/Post.tsx [hunk 3]
```diff
@@ -74,15 +77,15 @@ getUserDetailRequest(apiUrl)
           {currentPost.description}
         </div>
 
-        <RowComponent>
-          {/* {currentPost.LikedNum === 0 ?
+        {/* <RowComponent>
```
Comment (JordanLDavid) [inline src/components/Post/Post.tsx]: Cleanup code please. remove code that are commented out and unused.

Comment (TomYYHong) [reply src/components/Post/Post.tsx]: not sure is it useful for further enhancement

File: src/App.tsx (hunk 1 above)
Comment (TomYYHong) [reply src/App.tsx]: removed

File: src/components/Post/Post.tsx (hunk 2 above)
Comment (TomYYHong) [reply src/components/Post/Post.tsx:4]: removed

Comment (TomYYHong) [reply src/components/Post/Post.tsx]: not sure is it useful for further enhancement

File: .env.sample
Comment (511234) [inline .env.sample:1]: Restore .env.sample

Comment (TomYYHong) [reply .env.sample:1]: done

**Comment by TomYYHong** [comment]
image

**Review by yukfaihsu — APPROVED** [review]

File: src/components/Post/Post.tsx (hunk 3 above)
Comment (TomYYHong) [reply src/components/Post/Post.tsx]: cleaned

**Pull request merged**